src/dashboard/src/main/management/commands/rebuild_aip_index_from_storage_service.py

```diff
@@ -131,32 +131,36 @@ def handle(self, *args, **options):
         if options["uuid"]:
             package_uuid = options["uuid"]
             self.info("Reindexing package {}".format(package_uuid))
-            package_info = storageService.get_file_info(uuid=package_uuid)
+            try:
+                package_info = storageService.get_file_info(uuid=package_uuid)[0]
+            except IndexError:
+                self.error(
+                    "Error indexing package {0}. "
+                    "Package with supplied UUID not found in Storage Service".format(
+                        package_uuid
+                    )
+                )
```

Comment:
Is this `black` formatted?

Reply:
An `AttributeError` is mentioned in the PR comment, is this the error that we're covering here? Or is this a tangential issue?

Reply:
The `AttributeError` was being raised during `package_type = package_info.get("package_type")`, because `package_info` was the list returned by the Storage Service.

In this PR, I changed it so that `package_info` is the first item of the returned list: `package_info = storageService.get_file_info(uuid=uuid)[0]`. If there are no items in the list (i.e. a package with the specified UUID was not found in the Storage Service), it will raise an `IndexError`, so that's what I'm catching here.

I added some more information on the original bugs in this issue: https://github.com/archivematica/Issues/issues/1263

Reply:
The multi-line comment was run through `black`, but I had split the lines to try to keep line size reasonable. The changes made since have made that moot.

Reply:
Cool. It's just a little jarring as we have `Flake8` [ignoring line-lengths](https://github.com/artefactual/archivematica/blob/396015a44723e6b2dd2f4d7bed1443514a429c12/tox.ini#L38) (E501) up to a certain point to be [compatible with `black`](https://stackoverflow.com/a/59184533) so it's a bit more obvious when the line hasn't been auto-formatted.

Reply:
Yeah, fair!